claripy/backends/backend_z3.py

```diff
@@ -1,6 +1,7 @@
 
 import os
 import z3
+import gc
```

Comment:
Why do we need `gc`?

Reply:
@Phat3 you can get rid of the gc hack and replace the usage of the libz3 object with this: https://github.com/Z3Prover/z3/pull/2143